e2e-tests/content-strings/error-messages.js

```diff
@@ -119,6 +119,7 @@ export const ERROR_MESSAGES = {
             ABOVE_MAXIMUM: 'The maximum the buyer will owe cannot be more than £499.999. If you need cover for more than this, fill in this form instead.',
           },
         },
+        MULTIPLE: {},
```

Comment:
can we remove this if empty?

Reply:
Good shout but TBH in the next PR (coming shortly), this will be populated shall we leave this in for now?

Reply:
Removed it